etc/init.d/orchestrator.bash

```diff
@@ -26,6 +26,23 @@ SCRIPTNAME=/etc/init.d/$NAME
 # to increase this limit.
 ulimit -n 16384
 
+# initially noop but can adjust according by modifying orchestrator_profile
+# - see https://github.com/github/orchestrator/issues/227 for more details.
+post_start_daemon_hook () {
+	# by default do nothing
+	:
+}
+
+# Start the orchestrator daemon in the background
+start_daemon () {
+	# start up daemon in the background
+	$DAEMON_PATH/$DAEMON $DAEMONOPTS >> /var/log/${NAME}.log 2>&1 &
+	# collect and print PID of started process
+	echo $!
+	# space for optional processing after starting orchestrator
+	post_start_daemon_hook
```

Comment:
Is it not sufficient to add a comment into the existing function?
If not I can make another function call post_start_daemon_hook and redirect output there. It's a bit more convoluted but can be done if you want that.